### PR TITLE
Prevent $state.current to become empty afer refresh

### LINKED ISSUE #1345: (bug) User Management does not load user on page refresh
When you navigate to the user management page and click "Edit" on a user, the state+url changes to `/users/:id/edit` and a modal appears with all the user's data.  However, if you refresh from this page, the form loads empty, but still with the "update" label.  Submitting results in form validation rejecting the changes.

**Expected Result** 
The user's information should fill the form.

**Actual Result**
A blank form is loaded.

**Steps to reproduce**
 1. Navigate to `/users`
 2. Click `Edit` on any user.
 3. Wait for the modal to appear.
 4. Refresh the page using F5 or the refresh button on the browser.
 5. Observe an empty form appears.

### PR DESCRIPTION
**Pull request by lomamech**
Prevent $state.current to become empty afer refresh
- User Management
- Account Edit
- Billing Sercice Updare
- Inventory actions modal

closes #1345

### CODE REVIEW

File: client/src/modules/inventory/inventory.routes.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ angular.module('bhima.routes')
       })
       .state('inventory.create', {
         url : '/create',
+        params : {
+          creating : { value : true }
+        },
```
Comment (mbayopanda) [inline client/src/modules/inventory/inventory.routes.js:35]: What is the need of value ? What do you think about :
```
params : {
  creating : true,
},
```

**Review by mbayopanda — COMMENTED** [review]
LGTM

File: client/src/modules/inventory/inventory.routes.js (hunk 1 above)
Comment (lomamech) [reply client/src/modules/inventory/inventory.routes.js:35]: It makes it possible to differentiate the creation of the update, to consider only the present of the $state.params.uuid was not sufficient especially when one refreshed the browser

**Comment by jniles** [comment]
Seems like this got the green light!  Merging...

**Pull request merged**